__mocks__/swrv.js

```diff
@@ -1,11 +1,122 @@
+const baseURL = import.meta.env.VITE_TRACTION_BASE_URL
```

Comment:
I think the Vue 2.7 upgrade might actually mean we can use mirage here instead. But my timebox ran out.